sdk/keyvault/azure-keyvault-administration/setup.py

```diff
@@ -69,7 +69,7 @@
     python_requires=">=3.8",
     install_requires=[
         "azure-common~=1.1",
```

Comment:
Wondering if you still need azure-common. I know it's still necessary with some legacy autorest / multi-api, but it doesn't look like we do that here (might have missed it, though)

Reply:
We're no longer using legacy autorest or multi-API, but we are using `msrest` models so I naively assumed the dependency would still be necessary. Do you know if that configuration's safe without the dep?

Reply:
The generator doesn't list it as a necessary dependency: https://github.com/Azure/autorest.python/blob/main/packages/autorest.python/ChangeLog.md

I don't remember needing it until I generated with multiapi. But because she's already here, let's double-check with @iscai-msft 😸

Reply:
Checked with her offline and we're good to remove it! I'll apply that change to each library